tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1372,3 +1373,198 @@ def test_tiny_specaugment_librispeech(self):
         )
         # fmt: on
         self.assertTrue(torch.allclose(logits[0][0, 0, :30].cpu(), EXPECTED_LOGITS, atol=1e-4))
+
+
+def prepare_whisper_encoder_inputs_dict(
+    config,
+    input_features,
+    head_mask=None,
+):
```

Comment:
Fits on one line

Reply:
Resolved in https://github.com/huggingface/transformers/pull/21754/commits/fc8cb9a6bbe41395351d24178f222ae01a70758c